instrumentation/aws-sdk/aws-sdk-1.11/javaagent/build.gradle.kts

```diff
@@ -26,79 +26,91 @@ testSets {
   // in 1.11.106 than 1.11.84.
   // We test older version in separate test set to test newer version and latest deps in the 'default'
   // test dir. Otherwise we get strange warnings in Idea.
-  test_before_1_11_106 {
-    filter {
```

Comment:
These blocks move to Tests block since apparently groovy was resolving the `filter` method on the test task named `test_before_1_11_106` despite being nested in a configuration named `test_before_1_11_106` :O